src/components/Layout/NavLink/index.jsx

```diff
@@ -21,12 +23,24 @@ export default class NavLink extends Component {
   };
 
   render() {
-    const { children, linkPath, t } = this.props;
+    const { children, linkPath, user, t } = this.props;
     const paths = linkPath.split('>');
     const linkLen = paths.length - 1;
 
     const pathToLink = {
```

Comment:
separate links to external config file

Reply:
ok. pathToLink define to  external config file.